internal/cli/rules.go

```diff
@@ -106,7 +110,7 @@ func listRulesCmd(cli *cli) *cobra.Command {
 			if err != nil {
 				return err
 			}
-
+			cli.renderer.Warnf(rulesDeprecationLogText)
```

Comment:
Should we print these using a `PersistentPreRun` hook on the root `rules` command? That way it'll be the first thing printed once any rule command is run, and it will only have to be defined once.
See https://github.com/spf13/cobra/blob/main/site/content/user_guide.md#prerun-and-postrun-hooks

Reply:
Agreed, it's a good suggestion 👍🏻

Reply:
Definitely a good suggestion! I attempted to add on the root command through the `PersistentPreRun` hook but doing so incurs a pointer dereference whenever running one of the child commands. Might even be a bug in the Cobra framework. In lieu of this roadblock, are we ok with keeping the current solution? Considering this is a deprecated set of commands, it might not be worth sinking more time into.